doc/sphinx/users-guide/vcl-built-in-subs.rst

```diff
@@ -139,13 +139,45 @@ of the following keywords:
     see  :ref:`fail`
 
   ``lookup``
-    Look up the object in cache.
+    Control passes to :ref:`vcl_lookup` when coming from a ``hash``
+    return in `vcl_recv`.
```

Comment:
Shouldn't that be _when coming from vcl_hash_ ?

Reply:
Both technically.